sdk/Lusid.Sdk.Tests/tutorials/Instruments/FxOptionExamples.cs

```diff
@@ -105,7 +105,7 @@ public void FxOptionCreationAndUpsertionExample()
             _instrumentsApi.DeleteInstrument("ClientInternal", uniqueId);
         }
         
-        [LusidFeature("F10-3")]
+        [LusidFeature("F5-19")]
```

Comment:
In [features.yml MR](https://gitlab.com/finbourne/salesengineering/lusid-features/-/merge_requests/33/diffs) looks like this is described as "Create Equity Option". 
- L179